src/components/pages/Playlist.vue

```diff
@@ -1799,4 +1823,8 @@ h2 {
     }
   }
 }
+
+.load-more {
+  padding: 1em;
+}
```

Comment:
To reduce the global CSS files size, using `class="pa1"` can achieve the same goal.